python/src/aac/spec/spec.yaml

```diff
@@ -238,7 +246,6 @@ enum:
     - bool
     - date
     - file
```

Comment:
I'll have to think on Date and File since you can use/define those kinds of resources easily with primitives, which isn't something we could do with defining Map data structures and key/value types.